src/ripple/rpc/impl/ServerHandlerImp.h

```diff
@@ -42,69 +40,42 @@ bool operator< (Port const& lhs, Port const& rhs)
     return lhs.name < rhs.name;
 }
 
-class WSInfoSub : public InfoSub
+class ServerHandlerImp
+    : public Stoppable
```

Comment:
Yeah...well:
```
using ServerHandler = ServerHandlerImp;
```
This is just a down payment on a bigger refactoring. Getting these changes in will mean fewer conflicts later. Its a mechanical refactoring (no logic changes).